index.js

```diff
@@ -1,255 +1,269 @@
+'use strict'
```

Comment:
This really needs tests. Especially since there is a material change here

Reply:
Working on it